### PR TITLE
{math}[intel/2023a] AMICA v2024.1.19

### PR DESCRIPTION
**Pull request by pavelToman**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by pavelToman** [comment]
@boegelbot please test @ generoso

**Comment by pavelToman** [comment]
@boegelbot please test @ jsc-zen3

File: easybuild/easyconfigs/a/AMICA/AMICA-2024.1.19-intel-2023a.eb [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+
+source_urls = ['https://github.com/sccn/amica/archive/']
+sources = [{'download_filename': '%s.zip' % local_commit, 'filename': '%(name)s-%(version)s.zip'}]
+checksums = ['d856ef1bf4bd09b2aacf5da6832b9af5d7fb70976786f43585c72acd1a52443b']
+
+# original command: 
+# $ mpif90 -static-intel -fpp -O3 -march=core-avx2 -heap-arrays -qopenmp -mkl -DMKL -o amica17 funmod2.f90 amica17.f90
+cmds_map = [('.*', "$MPIF90 $F90FLAGS -static-intel -fpp  -heap-arrays -mkl -DMKL -o amica17 funmod2.f90 amica17.f90")]
+
+test_cmd = "./amica17nsg ./amicadefs.param"
```
Comment (boegel) [inline easybuild/easyconfigs/a/AMICA/AMICA-2024.1.19-intel-2023a.eb:21]: @pavelToman Please use `amica17` here, the binary that was built.

`amica17nsg` is a pre-built binary shipped in the repository, we don't want to use that for testing...

**Review by boegel — CHANGES_REQUESTED** [review]

**Comment by pavelToman** [comment]
@boegelbot please test @ generoso

**Review by boegel — APPROVED** [review]
lgtm

**Comment by boegel** [comment]
@boegelbot please test @ jsc-zen3

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
node3106.skitty.os - Linux RHEL 8.8, x86_64, Intel(R) Xeon(R) Gold 6140 CPU @ 2.30GHz, Python 3.6.8
See https://gist.github.com/boegel/4c2286a6642adac976b83f24150a7bfa for a full test report.

**Comment by boegel** [comment]
Going in, thanks @pavelToman!

**Pull request merged**